config/locales/en/budgets.yml

```diff
@@ -86,7 +93,9 @@ en:
           voted_html:
             one: "<strong>You voted one proposal with a cost of %{amount_spent}</strong>"
             other: "<strong>You voted %{count} proposals with a cost of %{amount_spent}</strong>"
-          voted_info: You can %{link} at any time until the close of this phase. No need to spend all the money available.
+          voted_info:
```

Comment:
Missing [view adaptation](https://github.com/consul/consul/blob/3d68d439a074abba389fc279bd1169f6567be5ea/app/views/budgets/ballot/_ballot.html.erb#L14) to new translation keys.

Reply:
Could you elaborate please. Do you mean this keys are not used?

Reply:
Ah yes I think this is already solved, I was referring to updating this key in the view https://github.com/consul/consul/pull/3199/files#diff-8549275dea93f8da3dd8c13ffbcad0f7R18